_posts/2020-09-23-cytech-customer-story.adoc

```diff
@@ -0,0 +1,45 @@
+
+The question is how. And this is the part that we entered the scene. The AI and machine learning algorithm was already proposed by a team of scientists. Its ambition was to cover exactly the need described above by efficiently choosing the right people to be tested. But these algorithms need data and at that point detailed data from the passengers entering the country were not available in any digital and organized form.
+
+
+== The basic idea
+The Greek government assigned our company, https://www.cytechmobile.com/[Cytech], to build a software that would gather the information from the passengers, filter it and pass it to all involved parties so they can do their job. One of them was the machine learning system called EVA. But the list with the parties needing this information is much longer: it includes the civil protection agency, the ministry of health, border agents, airlines etc. At the same time it would send an email to each of the passengers to inform them that their application is under review. When EVA would come up with the result it would send it back to us so we can email passengers the final document. Without it, their entrance in the country would not be permitted. To make a long story short, our job was to build a demanding https://travel.gov.gr[web form] to gather all passengers data but the most important part was to build a Data Broker capable of exchanging the right data with all the other information systems emphasizing on speed, robustness and resilience. 
+
+== Technical challenges and solutions
+The requirements of the Broker platform for the https://travel.gov.gr[Passenger Locator Form] project's backend mainly revolved around exposing REST API endpoints for the UI to consume, as well as interconnect with government-related 3rd parties for required data exchange, such as EVA, EODY (National Public Health Organization), GGPP (the General Secretariat for Civil Protection) and the Greek COVID19 patients registry. Also, in order to be delivered in line within the extreme deadlines we had, as well as being able to cover the large-scale needs of the project, we also required our tools to be mature and proven and provide us with developer productivity, (horizontal) scalability and maximum efficiency/performance with the minimum possible resources.
+
+We had positive experience with Quarkus in the past with small scale projects, but we were already impressed with its features for ease of use and increased productivity. Since Quarkus is based on industry standards (Microprofile and Jakarta EE apis), it was very easy for our developers to get up to speed very quickly and within the very tight deadlines. Also, the Guides section on Quarkus’s website really helped getting immediate feedback on the few parts that did not auto-magically work as expected.
```

Comment:
```suggestion
We had a positive experience with Quarkus in the past with small scale projects, but we were already impressed with its features for ease of use and increased productivity. Since Quarkus is based on industry standards (Microprofile and Jakarta EE APIs), it was straightforward for our developers to get up to speed very quickly and within the very tight deadlines. Also, the Guides section on Quarkus’s website helped to get immediate feedback on the few parts that did not auto-magically work as expected.
```